content/troubleshooting/net/GettingNetAgentLogs.md

```diff
@@ -8,7 +8,7 @@ In rare scenarios, bad instrumentation causes a web server process to crash or a
 
 ## Agent Logs Directory
 
-The .NET agent logs information to the *Contrast\dotnet\LOGS* directory within *C:\ProgramData\Contrast\dotnet\LOGS*, the Windows 2008/2012 *ProgramData* directory. Depending on the setup of the Windows profile and folder view settings, the directories may be hidden. If so, paste the paths into the Windows Explorer location; you may need to replace the drive letter *C* with *D*.
+The .NET agent logs information to the *LOGS* directory within *C:\ProgramData\Contrast\dotnet\*, the Windows 2008/2012/2016 *ProgramData* directory. Depending on the setup of the Windows profile and folder view settings, the directories may be hidden. If so, paste the paths into the Windows Explorer location; you may need to replace the drive letter *C* with *D*.
```

Comment:
We specify the C:\ProgramData\Contrast\dotnet\ then talk about `ProgramData` directory?

Reply:
Good idea. Switched.